### ISSUE TITLE
Initialize programmatically

### ISSUE THREAD

**Issue by nsslh** [dealfonso/power-buttons#9]
I'm having trouble integrating this into a Rails 7 app that uses bootstrap 5.

The button that I want confirmation on doesn't exist at document load time: it's added to the page later. How can I get power-buttons to notice the button and bind to it after document load time?

**Comment by dealfonso** [comment]
Hi,

in that case, you should use the programmatical way. E.g. add a script tag just after the buttons are created

```javascript
$('#mybutton').powerButtons('confirm', options);
```

or

```javascript
let mybutton = document.getElementById('mybutton');
window.powerButtons('confirm', mybutton, options);
```

But I think that it would be better to add a mechanism to re-launch the mechanism to inspect the DOM again. Let me check and I think it will be ready in the next few days

**Comment by dealfonso** [comment]
Hi,

I have added the posibility of discovering the plugins from the data-attributes programmatically by using the method `powerButtons` without parameters. If you want to trigger the discovery of new elements in the DOM, you can also use the function `powerButtons.discoverAll()`.

All these features are included in release [2.1.0](https://github.com/dealfonso/power-buttons/releases/tag/2.1.0)

Please try it and let me know
